meta/models.py

```diff
@@ -307,7 +317,7 @@ def get_absolute_url(self):
         return reverse('media_url', args=[str(self.id)])
 
     def is_video(self):
-        name, extension = os.path.splitext(self.file.name)
+        _, extension = os.path.splitext(self.file.name)
```

Comment:
Non-blocking issue, but any reason why `name` was changed to underscore?

Underscores shouldn't be used to avoid issues, since they are reserved for translations:

`from django.utils.translation import gettext_lazy as _`

Reply:
Sorry, I didn't realize that. I've changed it because `name` was not being used, but no problem going back on this.